translations/ui-organizations/en.json

```diff
@@ -273,6 +273,192 @@
   "data.countries.ZMB": "Zambia",
   "data.countries.ZWE": "Zimbabwe",
 
+  "data.languages.abk": "Abkhaz",
```

Comment:
o_O, shouldn't they be translated in common repo?

Reply:
In this case search in `FildSelection` doesn't work

Reply:
@AndreiShumski2 I tend to agree with Mikita, maybe it's better to have a common filter component, like `CountryFilter` instead of `SelectionFilter`. When we will have only one place with translations - in stripes-acq-components